tasks/04_pages/scraper.py

```diff
@@ -0,0 +1,83 @@
+import logging
+import requests
+import time
+
+from bs4 import BeautifulSoup as bs
+
+from common import sql, db_config
+
+
+def add_page_row(case_id: int, 
+                 case_number: str,
+                 raw_text: str):
+    
+    # insert relevant info to pages table in the db
+    try:
+        if db_config.db_type == 'sqlite':
+            query = '''INSERT INTO pages (case_id, case_number, raw_text)
+                            VALUES (?, ?, ?)
+                        '''
+        elif db_config.db_type == 'postgresql':
+            query = """INSERT INTO pages (case_id, case_number, raw_text) 
+                            VALUES (%s, %s, %s);
+                    """
+                
+        with sql.db_cnx() as cnx:
+            c = cnx.cursor()
+            c.execute(query, (int(case_id), case_number, raw_text))        
+            
+    except Exception as e:
+        print(f'Error adding page to {db_config.pages} table: {e}')
```

Comment:
You probably want to raise this exception in addition to printing, since if you're unable to write to the DB, you probably want the script to stop